traits.yml

```
@@ -146,3 +267,58 @@ traits:
             type: string
             description: >
               Contents of the token.
+specifications:
+  lifecycle:
+    description:
+      Specifications that are concerned with describing aspects of the
+      lifecycle of an entity, such as versioning.
+    members:
+      EntityVersionsRelationship:
+        description: >
+          Retrieves references to alternate versions of the same logical
+          entity. This may include unstable versions such as "latest".
```

[Consider]
"Retrieves references" reads to me as describing what happens when you use this relationship in openassetio, whereas a think a more declarative desciption is appropriate for this sort of definitional markup.

```suggestion
          A relationship between alternate versions of the same logical
          entity. This may include unstable versions such as "latest".
```